environment

```diff
@@ -104,36 +116,38 @@ DSS_AWS_FLASHFLOOD_PREFIX_READ_PROD="aws"
 DSS_AWS_FLASHFLOOD_PREFIX_WRITE_PROD="aws"  # This can be a comma separated list
 DSS_GCP_FLASHFLOOD_PREFIX_READ_PROD="gcp"
 DSS_GCP_FLASHFLOOD_PREFIX_WRITE_PROD="gcp"  # This can be a comma separated list
-DSS_INFRA_TAG_PROJECT=dcp
-DSS_INFRA_TAG_SERVICE=dss
-DSS_INFRA_TAG_OWNER=dss-team@data.humancellatlas.org
-DSS_NOTIFICATION_SENDER="DSS Notifier <dss.humancellatlas@gmail.com>"
-ADMIN_USER_EMAILS=""
-DCP_DOMAIN=${DSS_DEPLOYMENT_STAGE}.data.humancellatlas.org
-API_DOMAIN_NAME="dss.${DCP_DOMAIN}"
+
+DSS_INFRA_TAG_PROJECT="${DSS_PLATFORM}-dss"
+DSS_INFRA_TAG_SERVICE="dss"
+
+DSS_INFRA_TAG_OWNER="chmreid@ucsc.edu"
+DSS_NOTIFICATION_SENDER="chmreid@ucsc.edu"
+API_DOMAIN_NAME="dss.${DSS_DEPLOYMENT_STAGE}.ucsc-cgp-redwood.org"
+
 TOKENINFO_FUNC=dss.util.security.verify_jwt
 # TODO remove https://dev.data.humancellatlas.org/ from OIDC_AUDIENCE once seperate deployments are made
-OIDC_AUDIENCE=https://dev.data.humancellatlas.org/,https://${DCP_DOMAIN}/
+OIDC_AUDIENCE=https://auth.ucsc.ucsc-cgp-redwood.org
 AUTH_URL=https://auth.testing.data.humancellatlas.org
 OPENID_PROVIDER=https://humancellatlas.auth0.com/
 OIDC_EMAIL_CLAIM=https://auth.data.humancellatlas.org/email
 OIDC_GROUP_CLAIM=https://auth.data.humancellatlas.org/group
 NOTIFY_URL=https://${API_DOMAIN_NAME}/internal/notify
 DSS_AUTHORIZED_GOOGLE_PROJECT_DOMAIN_ARRAY=(
-    {human-cell-atlas-travis-test,broad-dsde-mint-{dev,test,staging}}.iam.gserviceaccount.com
+    platform-sc.iam.gserviceaccount.com
 )
 DSS_AUTHORIZED_DOMAINS=${DSS_AUTHORIZED_GOOGLE_PROJECT_DOMAIN_ARRAY[*]}
-DSS_AUTHORIZED_DOMAINS_TEST="human-cell-atlas-travis-test.iam.gserviceaccount.com"
-DSS_ES_DOMAIN="dss-index-$DSS_DEPLOYMENT_STAGE"
-DSS_ES_INSTANCE_TYPE="m4.large.elasticsearch"
+DSS_AUTHORIZED_DOMAINS_TEST="platform-sc.iam.gserviceaccount.com"
```

Comment:
I think this should be `platform-hca`, since this is the project ID and the project ID is still "project-hca"